challenge/src/components/announce/AnnouncementComponent.module.css

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
nit pick: we can be a bit more specific with this class name, for example `announcements container`